src/main/java/model/FruitOperation.java

```diff
@@ -0,0 +1,59 @@
+package model;
+
+public class FruitOperation {
```

Comment:
it looks like transaction not operation
```suggestion
public class FruitTransaction {
```